static/docs/user-guide/howto/index.md

```diff
@@ -0,0 +1,3 @@
+# How To
+
+- [Keep Data and Cache on External Drive](/doc/user-guide/howto/keep-data-and-cache-on-external-drive)
```

Comment:
See https://github.com/iterative/dvc.org/issues/727#issuecomment-545756811.

Reply:
I agree. But for the time being let's keep `index.md`, until the engine is fixed, and remove it later. It should not be difficult to remove it.

Reply:
Yes. You're right.